rct229/rules/section15.py

```diff
@@ -212,7 +212,7 @@ def get_calc_vals(self, context, data=None):
 
             return {
                 "baseline_transformer_efficiency": context.baseline["efficiency"],
-                "required_baseline_transformer_efficiency": table_8_4_4_lookup(
```

Comment:
Line 215 was fine before this latest change. What is needed is to access the `efficiency` field of the dictionary that is returned from `table_8_4_4_lookup(...)`. This is analogous to the `["lpd']` you used to access the `lpd` field of the dictionary returned from `table_G3_8_lookup(...)`.